### DISCUSSION TITLE
Request for new functionality: "Edit label line number"

### DISCUSSION THREAD

**Discussion by JoseCarlosGZ** [alefragnani/vscode-bookmarks#666] [Ideas]
Hello,

I am a frequent user of the VSCode Bookmark extension and I really appreciate the work you have done so far. I would like to suggest a new feature that I think could further improve the application.

I am explaining the case where we set a new label in the code, for example in line number 1000 of a particular file of our project, pointing to a specific statement that we want to have direct access.

If we later modify the code of this file by adding, for example, 50 new lines of code before where we have our label (line 1000), then this label will no longer point to the statement to which we had set it.

To solve this, instead of deleting the label and creating another one with the same name on line 1050, it occurs to me that it would be faster to edit the line number the label points to directly:

https://i.imgur.com/Mr5OxZT.png

PS. I appreciate your consideration and am happy to provide further details or clarification if required. Thank you for your time and hard work on Bookmarks 😊👍.

Kind regards, 
Jose Carlos

**Comment by alefragnani** [comment]
Hi @JoseCarlosGZ ,

First of all, sorry for the late.

Yes, this would be useful. I think not only the line number, but the position itself (line, column). 

Thanks for your suggestion

**Reply by alefragnani** [reply]
Created #678 to track
